scripts/crontab/crontab.tpl

```diff
@@ -7,6 +7,7 @@ HOME=/tmp
 
 # Every 5 minutes
 */5 * * * * %(django)s auto_approve
+*/5 * * * * %(django)s git_extraction
```

Comment:
We could consider using something like `1-60/5` for it not to start at exactly the same time as `auto_approve`, just to make it easier to follow logs? Maybe that's completely unnecessary though.